src/data/iea/IEA-APS-2024.json

```diff
@@ -77,7 +77,44 @@
+  "expertOverview": [
+    "#### Pathway Description",
```

Comment:
@jdhoffa @jacobvjk Before I sort out the linting errors and update the other data files, does this look like a bnicer file format?